docs/app/pages/components/components-sections/input-controls/radio-button/radio-button.component.html

```diff
@@ -94,4 +103,16 @@
     <ux-tab heading="TypeScript">
         <uxd-snippet language="javascript" [content]="snippets.compiled.appTs"></uxd-snippet>
     </ux-tab>
-</ux-tabset>
\ No newline at end of file
+</ux-tabset>
+
+<p>The following code can be used to create a reactive form:</p>
```

Comment:
I'm not keen on having a second set of code snippets that cannot be loaded in plunker. I think this can be summed up in the documentation with a sentence like

"In order to use the radio button in a reactive form, define the `formControlName` property on the `uxRadioButtonGroup` element."

I don't think this warrants a code snippet. Unless I am missing something?

Reply:
Added in text and removed snippet